script/testing/oltpbench/constants.py

```diff
@@ -23,7 +23,10 @@
 OLTP_DEFAULT_BIN = os.path.join(OLTP_GIT_LOCAL_PATH, "oltpbenchmark")
 OLTP_DEFAULT_COMMAND_FLAGS = "--histograms  --create=true --load=true --execute=true -s 5"
 OLTP_DIR_CONFIG = os.path.join(OLTP_GIT_LOCAL_PATH, "config")
+OLTP_DIR_CUR = os.getcwd()
 OLTP_DIR_TEST_RESULT = os.path.join(OLTP_GIT_LOCAL_PATH, "results")
+OLTP_DIR_TEST_SUMMARY_FOLDER = os.path.join(OLTP_DIR_CUR, "oltpsummary")
+OLTP_DIR_TEST_SUMMARY = os.path.join(OLTP_DIR_TEST_RESULT, "oltpbench.summary")
```

Comment:
Is OLTP_DIR_TEST_SUMMARY_FOLDER the destination folder where we are going to move the OLTP_DIR_TEST_SUMMARY file?

Reply:
Yes.  Then `archiveArtifacts 'build/oltpsummary/*.json'` will archive these files

Reply:
then it would be better to change the variable name from `OLTP_DIR_TEST_SUMMARY_FOLDER` to something like `OLTP_FILE_TEST_SUMMARY_FOLDER` to indicate it is a file, not a dir